test/e2e/reboots/kernel_parameter_remove.go

```diff
@@ -0,0 +1,157 @@
+package e2e
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+
+	coreapi "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	ntoconfig "github.com/openshift/cluster-node-tuning-operator/pkg/config"
+	nutil "github.com/openshift/cluster-node-tuning-operator/pkg/util"
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/util"
+)
+
+// Test the removal of a kernel parameter defined by a parent profile.
+var _ = ginkgo.Describe("[reboots][kernel_parameter_remove] Node Tuning Operator parent profile kernel parameter removal", func() {
```

Comment:
Maybe i missed that, but why not make this test file more general and include also additions/modifications of a kernel parameter (does'nt has to be in this PR)

Reply:
Good point, thanks!  Let's address this in a follow-up PR.  The two "reboots" tests are quite similar.  Perhaps they can be optimized/squashed together while reducing the number of reboots and extending the test coverage.

Reply:
Added scenario for adding kernel parameters in the child profile.  Marking as resolved for now.  Further improvements will be part of a follow-up PR if you agree.